### DISCUSSION TITLE
Possible to use Uno Platform with Unity 3D?

### DISCUSSION THREAD

**Discussion by sharpninja** [unoplatform/uno#12251] [Q&A]
Is it possible to use Uno Platform in a cross-platform Unity 3D project?

**Comment by jeromelaban** [comment]
Thanks for your question. 

At this time, this has not been a popular request and we do not have given many thoughts to implement it. Still, theoretically the Skia backend could be enhanced to render to any OpenGL environment, or using a environment similar to the Linux Framebuffer support.

What do you wish to be able to do with Uno in this context?

**Comment by sharpninja** [comment]
My goal would be to create games in Unity3D but create the UI with Uno Platform.  Most game UI libraries suck and/or very expensive.
